From 4d316646934b256da1086168167253e07eea5302 Mon Sep 17 00:00:00 2001
From: min zhang <zhangmin1984121@gmail.com>
Date: Thu, 20 Apr 2023 11:57:41 -0400
Subject: [PATCH] second time change

---
 alsamixer/cli.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/alsamixer/cli.c b/alsamixer/cli.c
index 5b0dc39..d850cde 100644
--- a/alsamixer/cli.c
+++ b/alsamixer/cli.c
@@ -42,6 +42,7 @@ static void show_help(void)
 {
 	puts(_("Usage: alsamixer [options]"));
 	puts(_("Usage: alsamixer [options] mjj first time"));
+	puts(_("Usage: alsamixer [options] mjj second time"));
 	puts(_("Useful options:\n"
 	       "  -h, --help              this help\n"
 	       "  -c, --card=NUMBER       sound card number or id\n"
